src/openms/source/ANALYSIS/MRM/ReactionMonitoringTransition.cpp

```diff
@@ -284,4 +284,36 @@ namespace OpenMS
     library_intensity_ = intensity;
   }
 
+  bool ReactionMonitoringTransition::isDetectingTransition() const
+  {
+    bool detecting = true;
+    if (this->getMetaValue("detecting_transition").toString() == "0") {detecting = false;}
```

Comment:
but that only works for true/false and not for 0/1, see https://github.com/OpenMS/OpenMS/blob/develop/src/openms/source/DATASTRUCTURES/DataValue.cpp#L670